### PR TITLE
Addition in resources section

### PR DESCRIPTION
**Pull request by Rishit-dagli**
Added Coursera course Machine Learning with TensorFlow on GCP

### CODE REVIEW

**Comment by Rishit-dagli** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -142,6 +142,7 @@ Build Type                                                        | Status
 *   [Getting Started with TensorFlow 2 from Coursera](https://www.coursera.org/learn/getting-started-with-tensor-flow2)
 *   [Intro to TensorFlow for Deep Learning from Udacity](https://www.udacity.com/course/intro-to-tensorflow-for-deep-learning--ud187)
 *   [Introduction to TensorFlow Lite from Udacity](https://www.udacity.com/course/intro-to-tensorflow-lite--ud190)
+*   [Machine Learning with TensorFLow on GCP](https://www.coursera.org/specializations/machine-learning-tensorflow-gcp)
```
Comment (kyscg) [inline README.md:145]:
```suggestion
*   [Machine Learning with TensorFlow on GCP](https://www.coursera.org/specializations/machine-learning-tensorflow-gcp)
```

**Review by kyscg — CHANGES_REQUESTED** [review]
Good addition :+1:

File: README.md (hunk 1 above)
Comment (Rishit-dagli) [reply README.md:145]: Thanks for pointing that out, seems like a typo

**Comment by Rishit-dagli** [comment]
@kyscg Thank you for pointing out the typo in the resources section, I have now resolved it.

**Review by mihaimaruseac — APPROVED** [review]

**Review by kyscg — APPROVED** [review]

**Review by shivaylamba — APPROVED** [review]

**Pull request merged**